Week1/hm/createDatabase.js

```diff
@@ -0,0 +1,14 @@
+const mysql = require('mysql');
+const connection = mysql.createConnection({
+    host: 'localhost',
+    user: 'hyfuser',
+    password : 'hyfpassword',
+})
+
+connection.connect();
```

Comment:
It's always a good idea to add validation on everything that could go on. And connecting to the DB is one of those places where tons of things can go wrong: user does not exist, wrong password, database does not exist, etc